### PR TITLE
ref(bug reports): remove archive functionality, resolve only

### PR DESCRIPTION
**Pull request by michellewzhang**
On the details, the buttons can look like:
SCR-20231026-jlww

or
SCR-20231026-jlon

On the list, there's a dropdown for mark read/unread but only one button for resolve/unresolve:

SCR-20231026-jpyw

### CODE REVIEW

**Comment by ryan953** [comment]
This PR could also go update the MailboxPicker and related things so we've just got the two options there too.

We should probably make the list query search for either `status: unresolved` or `!status: resolved` if possible. This way, if  that if someone were to visit the `/issues/` permalink for a thing, and were to click the archive button there... the feedback won't be missing from our interface.

**Comment by billyvg** [comment]
Do we need the dropdown menu at all since there's only one action now (resolve if currently unresolved, and vice versa)?

**Comment by michellewzhang** [comment]
> Do we need the dropdown menu at all since there's only one action now (resolve if currently unresolved, and vice versa)?

Good point! I'll update so we don't have dropdowns

File: static/app/components/feedback/decodeMailbox.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default function decodeMailbox(
     case 'resolved':
       return 'resolved';
     case 'archived':
-      return 'archived';
+      return 'resolved';
```
Comment (michellewzhang) [inline static/app/components/feedback/decodeMailbox.ts:12]: Is the logic... 
If mailbox = anything that isn't `unresolved` --> `resolved`? Or
If mailbox = anything that isn't `resolved` --> `unresolved`?

**Review by ryan953 — APPROVED** [review]

**Pull request merged**